README.tpl

```diff
@@ -1,5 +1,8 @@
 # {{crate}}
 
+[![Test Status](https://github.com/deshaw/srv-rs/workflows/Rust/badge.svg?event=push)](https://github.com/deshaw/srv-rs/actions)
+[![Crate](https://img.shields.io/crates/v/srv-rs.svg)](https://crates.io/crates/srv-rs)
```

Comment:
These should reference `{{crate}}` instead of `srv-rs` directly like the rest of the template does

Reply:
Thanks (and thanks for having CI set up to yell at me for missing the template in the first place).